EndpointHandler.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+require "settings.php";
+
+// Decodes the data array and returns the value of "claim"
+function getClaim($claim, $data) {
+	
+	$data_array = json_decode($data, true);
+	return $data_array[$claim];
+}
```

Comment:
Yes, do this. ^ If you don't this can conflict with any function named "getClaim" defined in any other plugin, theme or WordPress itself.

---
In reply to: [71198565](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71198565) [](ancestors = 71198565)